cv32e40s/docs/VerifPlans/Simulation/interrupts/CV32E40SX_CLIC.json

```diff
@@ -459,7 +459,7 @@
         "Pass/Fail Criteria": "N/A",
         "Test Type": "N/A",
         "Coverage Method": "N/A",
-        "Link to Coverage": ""
+        "Link to Coverage": "N/A - cannot take assertions in U-mode"
```

Comment:
Not a typo, clic interrupt priority is hard coded to machine mode and there are assertions that check that this signal tiefoff is not violated, thus this part of the clic spec is unreachable and unimplemented.

Reply:
What does it mean to _"take assertions in U-mode"_?

Reply:
Apologies, I am blind - cannot take INTERRUPTS in U-mode

Reply:
Note, I don't think it is worth fixing by itself, but if one for other reasons are going to edit the file I think this could be fixed at the same time.